vendor/bat-native-ads/src/bat/ads/internal/processors/behavioral/bandits/epsilon_greedy_bandit_processor.cc

```diff
@@ -45,21 +46,17 @@ targeting::EpsilonGreedyBanditArmMap MaybeAddOrResetArms(
     arm.value = kDefaultArmValue;
     arm.pulls = kDefaultArmPulls;
 
-    updated_arms[segment] = arm;
+    arms->insert_or_assign(segment, arm);
```

Comment:
can't you construct std::string here?
`arms->insert_or_assign(static_cast<std::string>(segment), arm);`

Reply:
std::string will be already constructed once during `find` call. So it would be optimal to reuse it.
It seems like, it is better to do std::move(string) here.